serverless-test-project/serverless.yml

```diff
@@ -1,5 +1,5 @@
 service: serverless-test-project
```

Comment:
Sorry, my YAML formatter decided to go fancy here. I can revert this if needed...